cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/seeds/SeedsManager.java

```diff
@@ -36,7 +36,7 @@
     private volatile ImmutableMap<String, DataCenterInfo> dataCenters;
     private final SchedulerClient client;
     private DefaultConfigurationManager configurationManager;
-    private StateStore stateStore;
+    private final StateStore stateStore;
```

Comment:
This member variable is never changed. It is being used to synchronize several blocks of code. It is a bad idea to synchronize on non-final fields (http://stackoverflow.com/questions/6910807/synchronization-of-non-final-field) and IntelliJ was giving a warning about it.